src/OmnichannelChatSDK.ts

```diff
@@ -639,7 +713,10 @@ class OmnichannelChatSDK {
     private async getIC3Client() {
         if (platform.isNode() || platform.isReactNative()) {
             this.debug && console.debug('IC3Core');
-            this.scenarioMarker.startScenario(TelemetryEvent.GetIC3Client);
+            this.scenarioMarker.startScenario(TelemetryEvent.GetIC3Client, {
```

Comment:
`GetIC3Client` is already done. See [here](https://github.com/microsoft/omnichannel-chat-sdk/blob/f9059fb6eec2623470efb30fce52e630a4a7d9dc/src/OmnichannelChatSDK.ts#L679)

It's not needed when platform is `Node` or `React Native`.

Reply:
Do we not need the RequestID or ChatID?

Reply:
No, `GetIC3Client` is used to the load the IC3 library. At this point in time, no chat has even started